test/runtests.jl

```diff
@@ -1,3 +1,3 @@
 using Test, ReTestItems, RAIRelTest
 
-include("helpers_test.jl")
+runtests("$(@__DIR__)/helpers_test.jl")
```

Comment:
I'm not sure if the `@__DIR__` is needed here. I think the relative path is fine because they are in the same directory?

Reply:
When I ran without it, I got `"No such path "helpers_test.jl"`, because julia resolves "helpers_test.jl" based on the "current working dir", which in CI (and usually locally as well) is the root of the repo.

Reply:
Quick question - is this `runtests` equivalent to wrapping the include in a `@testset`? If so, the macro looks cleaner to me. Feel free to go with whatever you prefer.

Reply:
it is! thanks, I agree that's a better way, I fixed it.